factcast-store-pgsql/src/test/java/org/factcast/store/pgsql/registry/http/HttpRegistryFileFetcherTest.java

```diff
@@ -53,8 +63,12 @@ public void testCreateSchemaUrl() throws Exception {
 
     @Test
     public void testNullContracts() throws Exception {
-        TestHelper.expectNPE(() -> new HttpRegistryFileFetcher(null));
-        TestHelper.expectNPE(() -> new HttpRegistryFileFetcher(null, new OkHttpClient()));
+        val uut = new HttpRegistryFileFetcher(baseUrl, client, registryMetrics);
```

Comment:
wondering, if this is still necessary to do with lombok using Objects.requireNonNull

Reply:
Yeah I was thinking about the same. Maybe skip for new things 🤔